go/vt/vtgate/evalengine/convert.go

```diff
@@ -39,6 +39,10 @@ type (
 )
 
 func (c *ConvertExpr) unsupported() {
+	throwEvalError(c.returnUnsupportedError())
+}
+
+func (c *ConvertExpr) returnUnsupportedError() error {
 	var err error
```

Comment:
nit: err variable can be removed.

Reply:
We need it. We write to it in the switch case. Otherwise we would need to return in each switch case. I don't hate that either.